can_tools/scrapers/official/SC/sc_demographics.py

```diff
@@ -81,3 +81,29 @@ def normalize(self, data: TableauScraper):
             )
             .query("value != -1")
         )
+
+        # sum over age and race columns to create independent demographic data
+        # creates dataframes for each demographic that smooths over the other demographics
+        # and concatenate these separate dfs into one.
+        # (e.g create a row like 12-15, all, all, all instead of 12-15, ai_an, all, all etc.)
+        dataframes = []
```

Comment:
I realized that this is a bit more complex for this scraper as it gives combinations of age, sex, and race (not just age and race),  so the logic needed to be expanded. 

This feels a little clunky, but I validated the outputs for Aiken County, and the values line up.